### ISSUE TITLE
ASAP Access To Alerts In Progress

### ISSUE THREAD

**Issue by Haselmaier** [Haselmaier/TrainAlert#2]
[ZoneMinder ](https://zoneminder.com/)is the open source software being used to analyze camera images and raise alerts based on image movement observed and where in the frame the movement is occurring.  

It is currently unclear when in the alerting process it is visible an alert is occurring.  For example, can an alert be seen only after the detected motion is complete?  Or is there a way to determine an alert has begun to be logged (but it is not yet completed) - in order to enable the train alert to be done as quickly as possible?

Jim will research - primarily through the ZoneMinder forums - to get more information on this question.

**Comment by mrjazzcat** [comment]
Thanks, Jim.  I will research [ZoneMinder](https://zoneminder.com/) and see what I can learn as well.  It will help if we both know something about the open-source code we are leveraging.

I will let you drive the model, since you have experience there.  One other question you might want to ask on the forums is whether they ZoneMinder team has ported their code to ARM.  We could use an rPI board or a BeagleBone as the device, if there code runs on ARM.  Porting to ARM isn't very complicated, so we might even be able to help them there.  I have a lot of experience with that.

**Comment by mrjazzcat** [comment]
Oops.  I should have corrected the typos :-)

**Comment by Haselmaier** [comment]
I posted the question in the ZoneMinder forum.  The thread can be found here:
https://forums.zoneminder.com/viewtopic.php?f=36&t=26370

**Comment by Haselmaier** [comment]
I've decided to abandon ZoneMinder for the time being.  The frequent-camera-disconnect issue has not been resolved.   Multiple questions posted in the ZoneMinder forum have resulted in no recommendations from the experts.  I experimented with some system settings to see if the issue could be resolved and these actions were not successful.

An alternative camera monitoring piece of software, iSpy, has been experimented with and shows strong promise.  A new Issue will be opened where status of migrating to that solution will be documented.

This issue will be closed.
